game_env.py

```diff
@@ -897,6 +897,73 @@ def play(self):
 
         return winner
 
+    def create_board_reps(self, transition_list):
+        """
+        Returns a dictionary containing transition list of each transformation of the board
```

Comment:
return a list as that will maintain a uniform communication framework across everything